package.json

```diff
@@ -78,6 +80,7 @@
     "skiafy-icons": "node ./src/scripts/gen-skia-icons.js",
     "check": "svelte-check --compiler-warnings a11y-click-events-have-key-events:ignore,a11y-no-static-element-interactions:ignore",
     "transform-tokens": "node ./src/tokens/transformTokens.js && tsc -p ./tsconfig-tokens.json",
+    "s2v": "mkdir -p ./tokens/android/drawable && npx s2v -f ./icons -o ./tokens/android/drawable",
```

Comment:
Optional: Could be worth making a small `js` script, like `gen-skia-icons`, but completely optional